### ISSUE TITLE
Update Khronos logo

### ISSUE THREAD

**Issue by kenrussell** [KhronosGroup/WebGL#2317]
In #2314 the WebGL logo was updated throughout the repository. The Khronos logo is out-of-date, too, and should be updated when there's time.

**Comment by jaibeer72** [comment]
What is the new logo?
I am trying to learn more and be a better contributor this issue seems simple enough maybe I can create a quick PR to fix this. 
and maybe also learn more?

**Comment by jaibeer72** [comment]
What is the new logo?
I am trying to learn more and be a better contributor this issue seems simple enough maybe I can create a quick PR to fix this. 
and maybe also learn more?

**Comment by kainino0x** [comment]
I think it's safe to assume the logo on [Khronos's website](https://www.khronos.org/) is the latest one.
https://www.khronos.org/assets/css/images/khronos-logo.svg

The main place it's used in WebGL is at the [top of the spec](https://www.khronos.org/registry/webgl/specs/latest/1.0/), which uses [this logo](https://github.com/KhronosGroup/WebGL/blob/master/resources/KhronosGroup-3D.png).

**Comment by kainino0x** [comment]
Thanks for looking into this!

**Comment by kenrussell** [comment]
Please note also that there are downloadable logo packs here:
https://www.khronos.org/legal/trademarks/

**Comment by kainino0x** [comment]
Good call, that's the right place to get the Khronos logo asset instead of the one I linked.

**Comment by jaibeer72** [comment]
All right I will get onto it tonight after work :D 
thanks for the information.
